python/tk_multi_publish2/progress/publish_logging.py

```diff
@@ -76,7 +76,7 @@ def emit(self, record):
             status = self._progress_widget.INFO
 
         # request that the log manager processes the message
-        self._progress_widget.process_log_message(record.msg, status, action)
+        self._progress_widget.process_log_message(record.getMessage(), status, action)
```

Comment:
Fixes an issue where `logger.debug("hello %", "world")` would log `hello` instead of `hello world`. `getMessage` will so the right thing and use the `%` operator.